rqd/rqd/cuerqd.py

```diff
@@ -137,7 +137,7 @@ def killFrame(self, frameId, message):
         self.frameStub.Kill(run_frame=runFrame, message=message)
 
 
-if __name__ == "__main__":
+def main():
```

Comment:
We should probably cleanup this whole bit later on. I added #491 to switch to using something like argparse.

Reply:
Agreed. I was tempted to just do it here but decided it's best to get tests in place first.